src/settings/PreferencesPage.jsx

```diff
@@ -258,6 +258,25 @@ const PreferencesPage = () => {
                   titleGetter={(it) => t(it.name)}
                   label={t('deviceSecondaryInfo')}
                 />
+                <FormControl>
+                  <InputLabel>{t('deviceShowAttribute')}</InputLabel>
+                  <Select
+                    label={t('deviceShowAttribute')}
+                    value={attributes.deviceShowAttribute || ''}
+                    onChange={(e) => {
+                      setAttributes({ ...attributes, deviceShowAttribute: e.target.value });
```

Comment:
Is there an easy way to clear this attribute when empty value is selected?

Reply:
In the existing code of the page, attributes are normally not removed when they are set with empty values; they are kept with an empty string or a default value. To maintain consistency, I manage empty values in a similar way:

Reply:
OK, I think it's ok to keep as is. It's a bit different though because most of the other attributes are basically expected to have something. This one is basically optional.